Jenkinsfile

```diff
@@ -20,35 +20,96 @@ pipeline {
+                                    sh 'mvn install -q -B -Powasp -DskipTests=true -Dpmd.skip=true -Dfindbugs.skip=true -Djacoco.skip=true -Dcheckstyle.skip=true -pl !libs/owasp-diff-runner,!distribution/docs'
+                                }
+                            }
+                        },
+                        sonarqube: {
+                            node('linux-large') {
+                                checkout scm
+                                withMaven(maven: 'M3', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings') {
+                                    withCredentials([string(credentialsId: 'sonarqube-token', variable: 'SONAR_TOKEN')]) {
+                                        sh 'mvn -q -B -Dfindbugs.skip=true -Dcheckstyle.skip=true org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar -Dsonar.host.url=https://sonarqube.com -Dsonar.login=$SONAR_TOKEN  -Dsonar.organization=codice -Dsonar.projectKey=ddf -pl !platform/solr/platform-solr-server-standalone,!distribution/docs,!distribution/test/itests/test-itests-ddf'
+                                    }
+                                }
+                            }
+                        },
+                        coverity: {
+                            node('linux-medium') {
+                                checkout scm
+                                withMaven(maven: 'M3', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings') {
+                                    withCredentials([string(credentialsId: 'ddf-coverity-token', variable: 'COVERITY_TOKEN')]) {
+                                        withEnv(["PATH=${tool 'coverity-linux'}/bin:${env.PATH}"]) {
+                                            configFileProvider([configFile(fileId: 'coverity-maven-settings', replaceTokens: true, variable: 'MAVEN_SETTINGS')]) {
+                                                echo sh(returnStdout: true, script: 'env')
+                                                sh 'cov-build --dir cov-int mvn -DskipTests=true -Dfindbugs.skip=true -Dcheckstyle.skip=true -Djacoco.skip=true install -pl !distribution/docs --settings $MAVEN_SETTINGS'
+                                                sh 'tar czvf ddf.tgz cov-int'
+                                                sh 'curl --form token=$COVERITY_TOKEN --form email=cmp-security-team@connexta.com --form file=@ddf.tgz --form version="master" --form description="Description: DDF CI Build" https://scan.coverity.com/builds?project=codice%2Fddf'
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        },
+                        nodesecurity: {
```

Comment:
✏️  Just to make it clear here that not referring to a Jenkins `node` here, should we call this `nodeJsSecurity` or something similar?